src/test/index.js

```diff
@@ -0,0 +1,37 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+// Define the expect assertion
+const { expect } = chai;
+
+// chai middleware
+chai.use(chaiHttp);
+
+describe('INDEX js', () => {
+  it('should get home successfully', (done) => {
```

Comment:
`should get home successfully` can be better rename to `should display a welcome message successfully`

Reply:
Okay cool, thanks

Reply:
done